scripts/cli_install.php

```diff
@@ -111,7 +114,7 @@ function getEnvVar(string $name, string $fallback = null, array $valid_values =
     'core' => [
         'hostname' => getEnvVar('NAMELESS_HOSTNAME', 'localhost'),
         'path' => getEnvVar('NAMELESS_PATH', ''),
-        'friendly' => getEnvVar('NAMELESS_FRIENDLY_URLS', 'false') === 'true' ? true : false,
```

Comment:
This was intentional, unless it magically works now for you? Gotta love php

Reply:
The result of === returns boolean true/false so this must work